packages/jest-worker/src/index.js

```diff
@@ -76,8 +76,13 @@ export default class {
     for (let i = 0; i < numWorkers; i++) {
       const worker = new Worker(workerOptions);
 
-      stdout.add(worker.getStdout());
-      stderr.add(worker.getStderr());
+      if (worker.getStdout()) {
+        stdout.add(worker.getStdout());
```

Comment:
Does V8 inline result of such function calls, or maybe it's better to create temporary variables?